lib/fog/ovirt/models/compute/template.rb

```diff
@@ -42,10 +42,12 @@ def ready?
 
         def destroy(options = {})
           service.client.destroy_template(id)
+        rescue ::OVIRT::OvirtException => e
```

Comment:
If the client is already wrapped, this should be unnecessary.

Reply:
removed it, didn't find any other locations beside `method_missing`